### ISSUE TITLE
Toke Social: A resort for cannabis connoisseurs

### ISSUE THREAD

**Issue by dafky2000** [decentraland/districts#32]
**Title:** Toke Social: A resort for cannabis connoisseurs
**Description:** An artistic style lounge for cannabis smokers to hang around, talk non-sense, and view the world from a different lens.
**Project Lead:** Dan Kelly (myself@danielkelly.me)
**Team:** Just me right now, "project manager" and programmer. Looking for people to fill 3D modelling, programming, community moderators/engagement, event planning/speaker acquisition, advertiser acquisition, and hidea (thinker) roles.
**Goal:** Mostly just for fun, maybe in the future we can acquire some advertising revenues if we do well.
**Category:** Public
**LAND Needed:** 100 - 400
**Startup Costs:** Community driven, cost of land minimum $2400USD (@0.024/MANA) 
**How will funds be raised:** By interest
**How will contributors see a return value:** Should we attract advertisers we would split the revenues appropriately - TBD as this is more just for fun but I would want financial and labour-based contributors to be compensated fairly.

# Design & Architecture
The Toke Social district's main building to be designed as a lounge with papers, vaporizers, bongs and pipes available for use. Meet new people around your favourite paraphernalia, get ripped, share reviews and stories.

Common hangout areas outside to feature trippy art and brain f***ery, maybe we could get some loaner art from the local DCL Art Museum (wink, wink, nudge, nudge). Would love to have enough area to carve out paths in a forest so users could go for a nice nature walk or ATV drive after their session and view the various organic matter from all over IRL - maybe another collaboration opportunity with a local DCL Science Center to show-and-tell some cool stuff to blow our minds.

I also hope to attract influential activists and speakers supporting marijuana.

# Selling points
 - Medical AND recreational cannabis use acceptable
 - No limits to how much marijuana you can carry or intake
 - No limits on where you can or cannot toke in Toke District
 - No impaired driving laws

# Revenue possibilities
In an effort to attract a larger crowd to the idea of Toke Social, here are some very initial proposed revenue possibilities.
 - Advertising (billboards, posters, product showcases/demos)
 - Sponsorship (events, speakers, contests)
 - Paid work (3D design, development, etc)

# In conclusion
This idea is a work in progress, contributions welcomed and appreciated - TIA! Personally, I have extremely little 3D/VR experience (only done a few Blender tutorials) but I am ambitious.

**Comment by finom-metall** [comment]
I love this idea and wanted to make something like that in small on my own land. But I don't have much knowlege with all this stuff and no idea how to build on my land.
I learned to design 3D models of machines in autodesk inventor but only for the metal industry and not for 3D gaming or whatever. So I don't think I can design the whole land but maybe some cool features. Right now I could offer a grinder, a working engine or a great telescope and I am definitely trying to make perfect 3D models of cannabis plants in every growing stage.

**Comment by dafky2000** [comment]
Awesome! Would also be cool to model various strains for some identification games. Could suppliment smell for descriptions of smells :-P

It looks like my contribution to this district has been reverted since the issues with terraform, but I am prepared to put up to half the minimum costs if we reach half, 50 LAND.

EDIT: Not sure what's going on with terraform yet but just logged and it says my 15 LAND is still allocated to TokeSocial :)

**Comment by dafky2000** [comment]
Sensimilla school ;-) Could have lots of learning stuffs too!

**Comment by finom-metall** [comment]
Identification could be quite difficult, even in real life it's not that simple... :D But we will see.
I'm excited how good the quality of the models can be. It would be cool to be able to zoom that close to a bud that you see the trichomes.

I would support this district with 10-15 LAND.
It is amazing how big the possibillities are when you think about it...

**Comment by dafky2000** [comment]
Added more details regarding revenue possibilities.
Added some more wanted roles.

**Comment by dafky2000** [comment]
I will update this thread with more details soon :)

**Comment by dafky2000** [comment]
There has been a lot of confusion with how districts will be handled. All in all there were a very limited number of investors in TokeSocial so the proposal will have to change quite a bit. I hope my intentions were clear that this was a donate to the cause relationship and land contributors will not have the direct opportunity to build here. 

If there are any profits made from this district, land owners and additional investors (additional funding and time contributors) will be compensated accordingly. This will need to be determined still.

I want to give anyone who misunderstood, the opportunity to build. I have won in the auction and will set aside roadside parcels to trade in exchange for your contribution share. I have no information about who contributed yet so please contact me if you think this is the route you'll want to take. Once I have the list, we can confirm and setup the transfer once supported.

I will be back with more information about possible new plans for the district with the new size restriction.

**Comment by dafky2000** [comment]
There has been a lot of confusion with how districts will be handled. All in all there were a very limited number of investors in TokeSocial so the proposal will have to change quite a bit. I hope my intentions were clear that this was a donate to the cause relationship and land contributors will not have the direct opportunity to build here. 

If there are any profits made from this district, land owners and additional investors (additional funding and time contributors) will be compensated accordingly. This will need to be determined still.

I want to give anyone who misunderstood, the opportunity to build. I have won in the auction and will set aside roadside parcels to trade in exchange for your contribution share. I have no information about who contributed yet so please contact me if you think this is the route you'll want to take. Once I have the list, we can confirm and setup the transfer once supported.

I will be back with more information about possible new plans for the district with the new size restriction.

**Comment by dafky2000** [comment]
**_Update: The offer still stands to trade misunderstood contributors a roadside land parcel for each parcel they contributed to TokeSocial. Contributors will not have the opportunity to directly build on or utilize TokeSocial property. They will at the very least have some voting rights in TokeSocial._**

TokeSocial will need to raise funds to build out, manage, promote and attract speakers our little paradise. I have been thinking about different ways to fairly build, distribute voting, decide who does what and who get shares/rights in TokeSocial for their work and how much. I have a few ideas but can't, in good conscience, come up with something and make that final decision on my own. It's not in the spirit of decentralization or a true democracy which is why I joined this project in the first place.

So far, only one contributor has reached out. He sold half of his contribution to another fellow, and they are both very excited to get this going and have contacts in the industry.

Please join the TokeSocial chatroom (https://chat.decentraland.org/channel/District_Tokesocial) and say hello or shoot me an email to the address at the top of this page in the original proposal. Let all decide on something together that works!

**Comment by dafky2000** [comment]
This discussion and planning has moved to https://github.com/dafky2000/tokesocial_proposals

**Comment by dafky2000** [comment]
I have uploaded a draft of the start-up proposal for review. We have good consensus among registered contributors already. Please get in touch if you have not registered your contribution!

https://docs.google.com/document/d/1qTs9cy27EED92U-GoQRkcr-3i3FYebPOZMZJXbvIncU/edit?usp=sharing

**Comment by Tiernn** [comment]
Hey, how is this going. I don't have much experience in this type of development. How do I get involved? There could be an angle to capitalise on what is happening in Canada and across the world in Medical! And advertising of course.

**Comment by dafky2000** [comment]
As per our successful startup plan, we needed to know of all contributors who plan to take part in the district and register within a timely manner. All contributors who did not register within 90 days after the vote do not qualify for any district benefits and have been refunded their original contributed MANA on Oct. 23 2018. If you believe you were refunded by mistake and you had registered, please reach out to discuss possibilities.
